hubspot/tasks.py

```diff
@@ -68,16 +74,66 @@ def check_hubspot_api_errors():
         defaults={"checked_on": now_in_utc()}
     )
     last_timestamp = hubspot_timestamp(last_check.checked_on)
+
     for error in get_sync_errors():
-        if error.get("errorTimestamp") > last_timestamp:
-            msg = "Hubspot error for {obj_type} id {obj_id}: {details}".format(
-                obj_type=error.get("objectType", "N/A"),
-                obj_id=error.get("integratorObjectId", "N/A"),
-                details=error.get("details", ""),
+        error_timestamp = error.get("errorTimestamp")
+        if error_timestamp > last_timestamp:
+            obj_type = (error.get("objectType", "N/A"),)
+            obj_id = parse_hubspot_id(error.get("integratorObjectId", "")) or "N/A"
+            error_type = error.get("type", "N/A")
+            details = error.get("details", "")
+
+            if (
+                "LINE_ITEM" in obj_type
+                and error_type == "INVALID_ASSOCIATION_PROPERTY"
+                and retry_invalid_line_association(obj_id, error_timestamp, details)
+            ):
+                continue
+
+            log.error(
+                "Hubspot error %s for %s id %s: %s",
+                error_type,
+                obj_type,
+                obj_id,
```

Comment:
```python
str(obj_id),
```

Reply:
The above changes are to avoid this exception being thrown:
```python
raised unexpected: ValueError("invalid literal for int() with base 10: 'N/A'",)
celery_1  | Traceback (most recent call last):
celery_1  |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 382, in trace_task
celery_1  |     R = retval = fun(*args, **kwargs)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 641, in __protected_call__
celery_1  |     return self.run(*args, **kwargs)
celery_1  |   File "/src/hubspot/tasks.py", line 89, in check_hubspot_api_errors
celery_1  |     and retry_invalid_line_association(obj_id, error_timestamp, details)
celery_1  |   File "/src/hubspot/tasks.py", line 121, in retry_invalid_line_association
celery_1  |     line = Line.objects.get(id=line_id)
```